chapter_convolutional-modern/resnet_vn.md

```diff
@@ -362,7 +365,11 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 5. Why cannot we just increase the complexity of functions without bound, even if the function classes are nested?
 -->
 
-*dịch đoạn phía trên*
+1. Tham khảo Bảng 1 trong bài báo :cite:`He.Zhang.Ren.ea.2016` để lập trình các biến thể khác nhau.
```

Comment:
```suggestion
1. Tham khảo Bảng 1 trong bài báo :cite:`He.Zhang.Ren.ea.2016` để thực thi các phiên bản ResNet khác.
```

Reply:
Em cảm ơn anh ạ. Hồi em học cấp 3 vs đh thì từ implement hay được dịch là `cài đặt` trong lập trình. Tuy nhiên mn ở đây lại thống nhất là `lập trình` nên chắc chúng ta thống nhất từ `lập trình`.

Reply:
cài đặt ---> installation 
implement --> thực thi, thực hiện 
code --> lập trình

Đây là từ không phải từ khóa. có thể dịch linh động để tránh nhàm chán. Em dịch một câu OK, nhưng đọc cả bài thì cảm giác nó lại khác. Nên linh động thay đổi mấy từ không quan trọng để cho hành văn tốt hơn.

Reply:
Dạ vâng ạ, e sẽ cố gắng dịch linh động lần sau. Em đc mấy bạn reviewer sửa lại nên dịch dịch sát các từ đó trong glossary. 
P/s: Các thầy VN mình hay có câu cài đặt thuật toán/chương trình được dịch từ cụm "implement...".

Reply:
@ngcthuong do từ `implement` rất linh động tùy theo bối cảnh nên khi nói đến việc implement một hàm hay một lớp thì bọn em đã thống nhất dịch là `lập trình` ạ.

Reply:
Từ chung chung này cũng cho vô glossary ah?

Reply:
Vâng, để thống nhất cách dịch của các bạn